### PR TITLE
Add powervr support

### PR DESCRIPTION
**Pull request by andrey-konovalov**

### CODE REVIEW

**Comment by koenkooi** [comment]
Looks good to me

File: conf/distro/include/egl.inc [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
 # Helper function for overloading the default EGL/GLES implementation.
 # The Mali libraries provided by ARM are compatible with the Mesa headers
 # and it is safe to use with user space applications linked against Mesa.
 
-def get_mali_handler(d, target):
+def get_egl_handler(d, target):
     """  Overloading the default EGL/GLES implementation."""
     features = d.getVar('MACHINE_FEATURES', True).split()
 
     mali_libs = d.getVar('MALI_USERLAND_LIBARIES', True);
```
Comment (koenkooi) [inline conf/distro/include/egl.inc]: @kuscsik

Comment (kuscsik) [inline conf/distro/include/egl.inc]: lgtm

Comment (andrey-konovalov) [inline conf/distro/include/egl.inc]: New version of the patchset:
https://github.com/96boards/meta-rpb/pull/37

**Pull request closed without merging**